Sebastian-Sole/handicappin
# ✨ feat: hole handicap strokes

## What was done
- Holes have handicap strokes added for max score calculation
- Fixed exceptional round calculation bug
- Added unit tests to hcp and round calculation functions

## How to test
- Add a round with strokes for a hole 10
- See that hcp strokes have been applied in round calculation holes table, and that max score is correctly calculated and adjusted

## Future work
- Refactoring and code clean up
